src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -285,6 +343,23 @@ public function getPath()
             $value = $closure($value);
         }
 
+        // resolve placeholder value
```

Comment:
useless comment imho

Reply:
consistent with other useless comments here :P this maintains the form of separation per task/code block.